src/js/components/UnitHealthDropdown.js

```diff
@@ -26,8 +28,8 @@ class UnitHealthDropdown extends React.Component {
     const items = keys.map(function(key) {
       return {
         id: key,
-        html: UnitHealthStatus[key].title,
-        selectedHtml: UnitHealthStatus[key].title
+        html: <Trans render="span" id={UnitHealthStatus[key].title} />,
```

Comment:
Oh, does it extract all the constants from that file `UnitHealthStatus`?

Reply:
Only if you add i18nMark to those strings. With this convention (no children inside Trans) nothing will be extracted here, but we know they are translated sine they were marked in constants/UnitHealthStatus

This is a simple example what natalie was referring to when she was talking about chasing down string use through component hierarchies. If UnitHealthStatus[key].title was passed down through a prop, it would need the same kind of wrapper when it finally gets rendered